Gemfile

```diff
@@ -24,31 +24,6 @@ def location_for(place, fake_version=nil)
   end
 end
 
-group :development, :unit_tests do
-  gem 'cisco_node_utils', '~> 1.0', require: false
-  gem 'rake', '~> 10.1.0',       require: false
-  gem 'rspec', '~> 3.1.0',       require: false
-  gem 'rspec-puppet',            require: false
-  gem 'mocha',                   require: false
-  gem 'puppetlabs_spec_helper',  require: false
-  gem 'puppet-lint',             require: false
-  gem 'pry',                     require: false
-  gem 'rubocop',                 require: false
-  gem 'simplecov',               require: false
-end
-
-beaker_version = ENV['BEAKER_VERSION']
-beaker_rspec_version = ENV['BEAKER_RSPEC_VERSION']
-group :system_tests do
```

Comment:
I removed it because that block of code isn't actually doing anything.  beaker, beaker-rspec, and serverspec are never installed from what I can tell, and beaker isn't used in our build system anyway.

```
root@puppetmaster:/home/robgrie/cisco-network-puppet-module# bundle install
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and installing your bundle as root will break this application
for all non-root users on this machine.
Using rake 10.1.1
Using CFPropertyList 2.2.8
Using ast 2.1.0
Using parser 2.2.3.0
Using astrolabe 1.3.1
Using net_http_unix 0.2.1
Using cisco_nxapi 1.0.0
Using cisco_node_utils 1.0.1
Using coderay 1.1.0
Using diff-lcs 1.2.5
Using docile 1.1.5
Using facter 2.4.4
Using json_pure 1.8.2
Using hiera 3.0.1
Using json 1.8.3
Using metaclass 0.0.4
Using method_source 0.8.2
Using mocha 1.1.0
Using powerpack 0.1.1
Using slop 3.6.0
Using pry 0.10.2
Using puppet 4.2.2
Using puppet-lint 1.1.0
Using puppet-syntax 2.0.0
Using rspec-support 3.1.2
Using rspec-core 3.1.7
Using rspec-expectations 3.1.2
Using rspec-mocks 3.1.3
Using rspec 3.1.0
Using rspec-puppet 2.2.0
Using puppetlabs_spec_helper 0.10.3
Using rainbow 2.0.0
Using ruby-progressbar 1.7.5
Using rubocop 0.34.2
Using simplecov-html 0.10.0
Using simplecov 0.10.0
Using bundler 1.10.6
Bundle complete! 12 Gemfile dependencies, 37 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
```